### DISCUSSION TITLE
Is there merit to making myMPD a Volumio/Moode plugin?

### DISCUSSION THREAD

**Discussion by ibex-are-goats** [jcorporation/myMPD#280] [Q&A] [answered]
First off, great work!

I don't like the Volumio/Moode interfaces. Behind the times compared to yours. I suspect this interface can run on an Rpi. Is there merit to making myMPD easily installable on those distros, perhaps like a Volumio plugin? Thanks for your answer :)

**Comment by jcorporation** [comment]
Thank you!

Rpi, DietPi and myMPD is a good combination. myMPD is in the optimized software list of DietPi.

I has no intention to integrate myMPD in Volumio or Moode. You should ask the developers of this distributions.

**Comment by ibex-are-goats** [comment]
Thank you for your quick reply! I'm just happy to inspire some discussion.

Have you asked around to see if other distro makers know about myMPD? Maybe someone will like to integrate it by default. Your work is very mature by now...

**Comment by jcorporation** [comment, marked as answer]
Thank you very much. Programming myMPD is only a hobby for me, so I prever to invest my time in making myMPD better and not for marketing. But if someone would make myMPD more popular, I am happy!
